src/main/java/core/model/HOModel.java

```diff
@@ -566,9 +564,10 @@ public final void removePlayer(Player player) {
      * save the model in the database
      */
     public final synchronized void saveHRF() {
+        var time = getBasics().getDatum().toDbTimestamp();
         DBManager.instance().saveHRF(getID(),
                 java.text.DateFormat.getDateTimeInstance().format(new java.util.Date(
-                        System.currentTimeMillis())), getBasics().getDatum());
+                        System.currentTimeMillis())), time);
 
         //basics
```

Comment:
I know they are not yours, but those comments could be deleted, they really don't provide a lot of value =)

Reply:
removed them